### ISSUE TITLE
V18

### ISSUE THREAD

**Issue by odehalaa** [Alplox/teles#7]
Hello,

Please I tried many time on latest version to use it but not loading the channels , and need to ask you if possible when version v18 will be available ?

Regards

**Comment by Alplox** [comment]
Can you share the error?

**Comment by odehalaa** [comment]
![image](https://github.com/Alplox/teles/assets/89680790/259cfdb6-0354-48f0-888a-ffb1d465d55a)

Many many thanks for your support and efforts , yes working on online but why when download it there is no channels and how we can add ifram channels from js please ?

**Comment by odehalaa** [comment]
really amazing efforts and amazing solutions to watch news 

please I have suggestion please to support and add 
playerjs https://playerjs.com ?
and if you can activate drag and drop from channel list ?

**Comment by odehalaa** [comment]
strange issue from version 17 and latest version when install the code and open it on multiple machines and same issue , no channels and nothing work

**Comment by Alplox** [comment]
> ![image](https://private-user-images.githubusercontent.com/89680790/330756261-259cfdb6-0354-48f0-888a-ffb1d465d55a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU3ODM3MzUsIm5iZiI6MTcxNTc4MzQzNSwicGF0aCI6Ii84OTY4MDc5MC8zMzA3NTYyNjEtMjU5Y2ZkYjYtMDM1NC00OGYwLTg4OGEtZmZiMWQ0NjVkNTVhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE1VDE0MzAzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZhNzNlM2VkMmI4Y2IwOTNiM2Q3OTdiZTNjYTIzNmI4M2QzY2MzM2I1OGE1MWYwNGEwYzM2ZjI4NGFlYThkYTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.9tjqv1AIfJQX5iOquzs_qJGx9Mt2d4LkS6yYvNqukOY)
> 
> Many many thanks for your support and efforts , yes working on online but why when download it there is no channels and how we can add ifram channels from js please ?

If you are just downloading the zip file from main an then doble clicking the index.html file you need to start a live server to it to work
Here's a guide on how to do it:
https://www.freecodecamp.org/news/vscode-live-server-auto-refresh-browser/

If you're already running it in a live server then check the console (F12) for the error.

**Comment by odehalaa** [comment]
yes many thanks now working on live server , really thanks for your efforts and please if you can add to support playerjs and iof you can support drag and drop please ?
